### PR TITLE
Modifies MakefragTest so that unit tests for RAMCloud can be ran.

### PR DESCRIPTION
**Pull request by OferMania**
Also adds make-ramcloud-test script for dev env shell.
There's still test flakiness to resolve in follow-up PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +132,14 @@ perform an incremental build. If you need to make a clean build you can just
 delete the RAMCloud directory and the contents of the RAMCloud-install directory
 and run the above commands again to rebuild it.
 
+# Debugging
```
Comment (ghost) [inline README.md:135]: Where to place core files is personal preference and not something that I think belongs in the README.

File: patches/series [hunk 2]
```diff
@@ -4,3 +4,4 @@ fix-dpdk-mbuf-release.patch
 dpdk-config.patch
 multiop-includes.patch
 remove-java.patch
+tickets
```
Comment (ghost) [inline patches/series]: tickets doesn't seem like a meaningful name, and doesn't fit with the rest of the patches. `make-rc-test.path` or something similar seems better.

**Review by ghost — CHANGES_REQUESTED** [review]

File: patches/series (hunk 2 above)
Comment (OferMania) [reply patches/series]: Renaming to match convention of other patches.

File: README.md (hunk 1 above)
Comment (OferMania) [reply README.md:135]: I'd rather have this covered in the README, and is likely useful to have when working with RAMCloud.

Comment (vexingcodes) [reply README.md:135]: We don't have to give an actual value for `kernel.core_pattern`; instead, we can just say you may need to change this, you should look at its value to see what it is, and point them at the [man page](http://man7.org/linux/man-pages/man5/core.5.html) which explains how different values work and how the '%' placeholders are expanded.

Comment (OferMania) [reply README.md:135]: done

**Review by vexingcodes — APPROVED** [review]

**Pull request merged**